tests/suites/test_suite_pkcs7.function

```diff
@@ -0,0 +1,399 @@
+/* BEGIN_HEADER */
+#include "mbedtls/bignum.h"
+#include "mbedtls/pkcs7.h"
+#include "mbedtls/x509.h"
+#include "mbedtls/x509_crt.h"
+#include "mbedtls/x509_crl.h"
+#include "mbedtls/oid.h"
+#include "sys/types.h"
+#include "sys/stat.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_PKCS7_C:MBEDTLS_FS_IO:MBEDTLS_RSA_C:MBEDTLS_SHA256_C:MBEDTLS_SHA512_C:MBEDTLS_SHA1_C
```

Comment:
Please don't put dependencies on the test suite as a whole if they only apply to specific functions or test cases. People use Mbed TLS in a wide variety of configurations, so it's important to have good test coverage in a wide variety of configurations. Testing PKCS#7 only if RSA and stdio and several hashes are available is not sufficient. The only dependency that should be for the whole test suite is `MBEDTLS_PKCS7_C`.